src/main/cms/cms_menu_imu.c

```diff
@@ -280,6 +282,7 @@ static CMS_Menu cmsx_menuProfileOther = {
     .entries = cmsx_menuProfileOtherEntries,
 };
 
+/*
```

Comment:
Probably also temporary to what condition. In case it is forgotten when it should be re-enabled, as a safeguard in the case it is forgotten.